src/popup/router/components/Plate.vue

```diff
@@ -26,7 +26,7 @@
 
   &.only-top,
   .top {
-    background-color: variables.$color-bg-3-new;
+    background-color: variables.$color-bg-3;
```

Comment:
Same here, I guess the Plate should inherit the color from current screen with the use of `--screen-bg-color`.